internal/tools/aws/constants.go

```diff
@@ -108,6 +108,9 @@ const (
 	// DefaultAWSClientRetries supplies how many time the AWS client will retry a failed call.
 	DefaultAWSClientRetries = 3
 
-	// DefaultScheduledEncryptionKeyDeletion is the default number of days that AWS will take to delete an encryption key.
-	DefaultScheduledEncryptionKeyDeletion = 45
+	// KMSMaxTimeEncryptionKeyDeletion is the maximum number of days that AWS will take to delete an encryption key.
+	KMSMaxTimeEncryptionKeyDeletion = 30
+
+	// KMSMinTimeEncryptionKeyDeletion is the minimum number of days that AWS will take to delete an encryption key.
+	KMSMinTimeEncryptionKeyDeletion = 7
```

Comment:
I don't see these used anywhere. Is that intended?

Reply:
Sorry, meant I don't see the first `KMSMaxTimeEncryptionKeyDeletion` value used anywhere.

Reply:
The intent is to be used in the same way that KMSMinTimeEncryptionKeyDeletion but I guess we can delete it if not being used.

Reply:
I just wanted to make sure. Thanks